db/migrate/20200428102014_add_bulk_invite_link_to_invites.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddBulkInviteLinkToInvites < ActiveRecord::Migration[6.0]
+  def change
+    add_column :invites, :max_redemption_allowed, :integer, null: false, default: 1
+    add_column :invites, :redemption_count, :integer, null: false, default: 0
+    add_column :invites, :invite_link_expires_at, :datetime, null: true
```

Comment:
does this mean invites have 2 expirys?

Reply:
Yes, normal invites expires based on `invite_expiry_days` site setting. Invite Links can different expiry dates based on user input at the time of link create. Here is the method that decides if invite has expired based on invite type: https://github.com/discourse/discourse/pull/9813/files#diff-9592cb71c2bb6ae748d9ac2fc3596e86R66

Reply:
sorry ... what I meant ... why not call this column `expires_at` ? `invite_link_expires_at` is a bit strange.

Reply:
also ... we could either leave this as null and defer to site setting, or set it in stone on creation.

Reply:
I'll rename `invite_link_expires_at` with `expires_at`. Noted regarding the options for single use invite.

Reply:
Renamed `invite_link_expires_at` to `expires_at`.